[MWAN Conn] Serial console (#20001)
cloudflare/cloudflare-docs: src/content/partials/magic-wan/connector/configure-connectors.mdx

## DHCP vs static IP connections

{ props.magicWord === "hardware" && (
  <>

>     <p>At first boot, Magic WAN Connector needs to reach out to Cloudflare to download your settings and go through the activation process. If any of the networks plugged into your Connector are DHCP enabled, do not use a VLAN, and have an Internet connection, that process is handled automatically. However, if all of the networks require more information to utilize, (such as a network with static IPs, or tagged VLAN networks) your Magic WAN Connector might need some more information to proceed. There are couple of ways to provide that information. Feel free to choose the one that fits your workflow: </p>

Reviewer: ```suggestion
    <p>At first boot, Magic WAN Connector needs to reach out to Cloudflare to download your settings and go through the activation process. If any of the networks plugged into your Connector are DHCP enabled, does not use a VLAN, and have an Internet connection, that process is handled automatically. However, if all of the networks require more information to utilize, (such as a network with static IPs, or tagged VLAN networks) your Magic WAN Connector might need some more information to proceed. There are couple of ways to provide that information. Choose the one that fits your workflow: </p>
```
I'm not sure on the grammar here, but "do not use a VLAN" can be read as an instruction, which could be confusing.

  )
}

{ props.magicWord === "hardware" && (
  <>
		<AnchorHeading title="Bootstrap via Serial Console" depth={2} />

> 		<p>Advanced users can locally configure their Magic WAN Connector to work in a static IP configuration. This local method does not require having access to a DHCP Internet connection. However it does require being comfortable with using tools to access the serial port on Magic WAN Connector as well as using a serial terminal client to access the Connector's environment.</p>

Reviewer: ```suggestion
		<p>Advanced users can locally configure their Magic WAN Connector to work in a static IP configuration. This local method does not require having access to a DHCP Internet connection. However, it does require being comfortable with using tools to access the serial port on Magic WAN Connector as well as using a serial terminal client to access the Connector's environment.</p>
```

		<ul>
			<li>The Magic WAN Connector appliance</li>
			<li>A Phillips-head screwdriver</li>
			<li>A micro-USB to USB-A cable (there should be one included in the packaging of your Connector)</li>
			<li>A computer with an available USB port</li>
			<li>A serial terminal client</li>
			<li>Optional: if needed, a USB-A to USB-C converter dongle if your computer requires it</li>
		</ul>

		<AnchorHeading title="1. Access Magic WAN Connector's serial port" depth={3} />
		<ol>
			<li>Using the Phillips screwdriver, loosen the screw covering the serial console panel on the back of the Magic WAN Connector and turn the panel out of the way.
				<ol>
						<li>Pictures and more instructions can be found on <a href="https://www.dell.com/support/kbdoc/en-us/000134440/how-to-access-console-port-of-dell-emc-networking-virtual-edge-platform-1405-series" target="_blank" rel="noopener noreferrer">Dell's Technical Documents</a>.</li>
				</ol>
			</li>
			<li>Connect your computer to the Magic WAN Connector using the USB cable.</li>
		</ol>

		<AnchorHeading title="Default password to access Connector" depth={4} />

> 		<p>Your Connector's default password is the serial number (also known as a Service Tag for Dell devices), all uppercase plus an "!" (for example, <code>A1B2C3D!</code>)</p>

Reviewer: ```suggestion
		<p>Your Connector's default password is the serial number (also known as a Service Tag for Dell devices), all uppercase followed by an "!" (for example, <code>A1B2C3D!</code>)</p>
```
